### PR TITLE
Nicolas/opentelemetry

### PR DESCRIPTION
**Pull request by callicles**
* Removes sentry for observability of the CLI (not used anyway)
* Swich the HTTP backend for the logger from hyper to reqwest
* Improves inserter logs
* Updates all the open telemetry dependencies

### CODE REVIEW

File: apps/framework-cli/src/utilities/capture.rs [hunk 1]
```diff
@@ -88,6 +90,13 @@ pub struct MooseActivity {
     pub ip: Option<String>,
 }
 
+fn serialize_uuid<S>(id: &Uuid, serializer: S) -> Result<S::Ok, S::Error>
```
Comment (phiSgr) [inline apps/framework-cli/src/utilities/capture.rs]: a comment or a more descriptive function name why we need this specific function e.g. `serialize_uuid_as_str` `// serialized as_bytes by default for some serializers`

File: apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs [hunk 2]
```diff
@@ -140,8 +141,10 @@ impl<C: ClickHouseClientTrait + 'static> Inserter<C> {
                         }
                     }
                     Err(e) => {
-                        error!("Error inserting records to {}: {:?}", table, e);
-                        debug!("Failed batch size: {}", batch_size);
+                        error!(
+                            "Error inserting records {} to {}: {:?}",
+                            batch_size, table, e
```
Comment (phiSgr) [inline apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs]: The `{}` for `batch_size` should go before "records"?

File: apps/framework-cli/src/cli/logger.rs [hunk 3]
```diff
@@ -96,7 +95,25 @@ pub struct LoggerSettings {
...
+    match s {
+        Some(s) => match s.parse() {
+            Ok(uri) => Ok(Some(uri)),
+            Err(e) => {
+                error!("Failed to parse URI: {}", e);
+                Ok(None)
+            }
+        },
+        None => Ok(None),
+    }
```
Comment (phiSgr) [inline apps/framework-cli/src/cli/logger.rs]: `Ok(s.and_then(...))`?

File: apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs (hunk 2 above)
Comment (callicles) [reply apps/framework-cli/src/infrastructure/olap/clickhouse/inserter.rs]: English 😏

**Review by phiSgr — APPROVED** [review]

**Pull request merged**